### PR TITLE
Fix window path separators

### PR DESCRIPTION
**Pull request by JeanRemiDelteil**
Fix issues linked to system dependant path separators ("/" or "\")

### CODE REVIEW

File: src/transform/bare-specifiers.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const bareToPathSpecifiersTransform = (): Transform =>
       const scriptSource = await getFileContents(file);
 
       try {
-        const pathParts = file.path.split('node_modules/');
+        const pathParts = file.path.split('node_modules');
```
Comment (Cypher1) [inline src/transform/bare-specifiers.ts:32]: Is the logging this is used in particularly helpful? 
#7 is attempting to remove it.
Maybe we need to add some kind of verbosity flag?

Comment (JeanRemiDelteil) [reply src/transform/bare-specifiers.ts:32]: is this not used anywhere else then in the login ?
If that is not the case, then it was only to fix the log output, and avoid misleading users

Comment (Cypher1) [reply src/transform/bare-specifiers.ts:32]: I think it's only used in the logging. 
e.g. https://ci.appveyor.com/project/arcs/arcs-3i77k/builds/19992906

There's a lot of extra output just for this one log.

**Comment by Cypher1** [comment]
Sorry @JeanRemiDelteil, did we resolve this?

**Comment by JeanRemiDelteil** [comment]
@Cypher1 I'm not sure, this is not merged, so the issue with window path separator must remain if not fixed elsewhere.

I'm not using empathy anymore: a build using webpack is in place now.

**Pull request closed without merging**